streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -404,8 +404,8 @@
         "<code>org.apache.kafka.common.serialization.Serde</code> interface.";
 
     public static final String WINDOWED_INNER_CLASS_SERDE = "windowed.inner.class.serde";
-    private static final String WINDOWED_INNER_CLASS_SERDE_DOC = " Default serializer / deserializer for the inner class of a windowed record. Must implement the \" +\n" +
-        "        \"<code>org.apache.kafka.common.serialization.Serde</code> interface.. Note that setting this config in KafkaStreams application would result " +
+    private static final String WINDOWED_INNER_CLASS_SERDE_DOC = " Default serializer / deserializer for the inner class of a windowed record. Must implement the " +
+        "<code>org.apache.kafka.common.serialization.Serde</code> interface. Note that setting this config in KafkaStreams application would result " +
```

Comment:
Nice catch! Note, current doc is like this:
![image](https://user-images.githubusercontent.com/43372967/157572498-c1200a0f-ac13-4f8d-b991-7d2f2e2e1d11.png)